pkg/buffer/buffer.go

```diff
@@ -329,12 +339,21 @@ func (b *Buffer) calc(pkt []byte, arrivalTime int64) {
 	}
 	b.lastTransit = transit
 
-	if b.tcc {
+	if b.twcc {
 		if ext := p.GetExtension(b.twccExt); ext != nil && len(ext) > 1 {
 			b.feedbackTWCC(binary.BigEndian.Uint16(ext[0:2]), arrivalTime, p.Marker)
 		}
 	}
 
+	if b.audioLevel {
+		if e := p.GetExtension(b.audioExt); e != nil && b.onAudioLevel != nil {
+			ext := rtp.AudioLevelExtension{}
+			if err := ext.Unmarshal(e); err == nil {
+				b.onAudioLevel(ext.Level)
+			}
+		}
```

Comment:
For GetExtension and Unmarshal could we log the error at debug level. That could be really helpful for debugging.

If there is an error I think we should call `onAudioLevel` with 127. Otherwise we might think a user is still making noise, when instead they're having network problems.

Reply:
If we log the error, we are going to get crazy errors logging there, the only way we could get an error there if it's for some reason the ext map ID is wrong.

It's not neccesary to send 127 since if value is wrong is not going to get registered.